config/application.rb

```diff
@@ -41,5 +41,8 @@ class Application < Rails::Application
       Sequel::Model.db.extension :server_block
       Sequel::Model.db.extension :auto_literal_strings
     end
+
+    config.sequel.allow_missing_migration_files = \
+      (ENV['ALLOW_MISSING_MIGRATION_FILES'].to_s == 'true')
```

Comment:
Out for curiosity, will this have the same effect as `rake db:migrate_without_missing"`